src/hotspot/share/gc/parallel/vmStructs_parallelgc.hpp

```diff
@@ -57,7 +57,6 @@
   nonstatic_field(PSYoungGen,                  _min_gen_size,                                 const size_t)                          \
   nonstatic_field(PSYoungGen,                  _max_gen_size,                                 const size_t)                          \
                                                                                                                                      \
-  nonstatic_field(PSOldGen,                    _reserved,                                     MemRegion)                             \
```

Comment:
Can you please test tier4/5 because of the removal of this `vmStructs` field to make sure that the SA agent still runs? The tests for it are in tier4 (or 5).

Reply:
Tier4-6 pass.